sensor/AdblockPlugin.js

```diff
@@ -78,7 +80,16 @@ class AdblockPlugin extends Sensor {
 
     async apiRun() {
       extensionManager.onSet("adblockConfig", async (msg, data) => {
-        await rclient.setAsync(configKey, JSON.stringify(data));
+        const originConfig = await rclient.hgetAsync(configKey, "adblock");
+        if (originConfig != null) {
+          const originConfigObj = JSON.parse(originConfig);
+          for (const field in data) {
```

Comment:
this means it will never be able to override any fields that are NOT passed by the app. Why not just simply overriding the redis key with data from app?